### ISSUE TITLE
DON'T REMOVE version info from files!

### ISSUE THREAD

**Issue by Hoffi1** [jemproject/JEM-Project#1023]
I though it was clear NOT to remove the `@version`line from php/ini files.
That's the last time I talk about that. Next time I stop my work.

**Comment by JKoelman** [comment]
> I though it was clear NOT to remove the @versionline from php/ini files.

sorry for that Hoffi, but it was not clear to me. Seems you rather want it to have so fine with me to leave it.  And you know you could have send me a pm in the forum.

**Comment by Heklaterriol** [comment]
Ops, that is what we did recently

**Comment by mckillo** [comment]
We can apologize for the change. 

We don't know the reasons why the version was included, but in the development of other extensions and shared code, it doesn't make sense in a continuous development process to change a reference text in all files that doesn't add anything to the code, especially when GitHub tracks the changes. If a file hasn’t changed, why version it with the global version? And if it has changed, why still version it with the global version? This kind of change means that when you want to compare versions od package, all files appear modified, making it hard to see which files really changed between versions, and you have to apply filters to ignore comments. If you need to see changes in files or specific lines, GitHub already has the Blame feature.

In summary, I believe that not including the version makes version control easier. It's an unnecessary change at this point, same tiem and maybe it served another purpose back in the day. We're adapting, and those who come after us will do the same. 

Thanks to those who came before us for their work and for helping to continue the extension.

This issue could be closed.
